### PR TITLE
Add trivy.githubToken config key to the starboard ConfigMap

### LINKED ISSUE #43: 403 API rate limit exceeded when running starboard find vulnerabilities command
Trivy downloads `trivy.db` file from GtiHub release page which is subject to rate limit. Beyond that we do not cache the DB file.

2020-06-03T08:45:29.803Z INFO Need to update DB
2020-06-03T08:45:29.803Z INFO Downloading DB...
2020-06-03T08:45:29.962Z FATAL failed to download vulnerability DB: failed to download vulnerability DB: failed to list releases: GET https://api.github.com/repos/aquasecurity/trivy-db/releases: 403 API rate limit exceeded for X.X.X.X. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) [rate reset in 32m22s]

### PR DESCRIPTION
**Pull request by mozillazg**
Closes #43

### CODE REVIEW

File: pkg/find/vulnerabilities/trivy/scanner.go [hunk 1]
```diff
@@ -152,6 +152,18 @@ func (s *Scanner) PrepareScanJob(_ context.Context, workload kube.Object, spec c
 						},
 					},
 				},
+				{
+					Name: "TRIVY_GITHUB_TOKEN",
```
Comment (danielpacak) [inline pkg/find/vulnerabilities/trivy/scanner.go:156]:
```suggestion
					Name: "GITHUB_TOKEN",
```

I believe it's `GITHUB_TOKEN` as documented in https://github.com/aquasecurity/trivy#github-rate-limiting

**Review by danielpacak — CHANGES_REQUESTED** [review]
Thank you @mozillazg for the PR. It looks good, but we have to change the name of the environment variable so it's actually picked by Trivy

File: pkg/find/vulnerabilities/trivy/scanner.go (hunk 1 above)
Comment (mozillazg) [reply pkg/find/vulnerabilities/trivy/scanner.go]: Fixed.

**Review by danielpacak — APPROVED** [review]
LGTM!

**Pull request merged**